.github/workflows/main.yml

```diff
@@ -28,6 +28,10 @@ jobs:
 
     - name: Test with pytest
       run: |
+
+        export PATH=$PATH:$(pwd)
```

Comment:
Can you explain a little bit why this is here?

Reply:
I'm creating a sbatch script echoing "submitting job 1234"

This is enough to make slurmpy happy.

The export thing is used to add the directory of the "sbatch" script to path:
in doing so, any program calling on sbatch will get a nice and familiar reply:

"submitting job 1234"